pom.xml

```diff
@@ -434,12 +434,12 @@
           <!-- init Highlight -->
           <additionalOptions>${javadocAdditionalOptions}</additionalOptions>
           <nohelp>true</nohelp>
-          <show>private</show>
```

Comment:
I think what matters most is to expose the documentation of `public` and `protected` elements, which is the default if this parameter is not configured.

Reply:
agreed